tests/airportDetailsRetriever.test.js

```diff
@@ -0,0 +1,17 @@
+const { getAirportInformation } = require('../src/airportDetailsRetriever');
```

Comment:
Better to mock `const airportDetails = require('../resources/airportsInfo.json');` as test shouldn't be dependent on values in the .json file, what if `MAS` is removed permanently from file or `FFS` gets added to file. The tests would fail